### PR TITLE
Nvyn

### PR DESCRIPTION
**Pull request by NazarVynnyk**
Add StoresProducts interface and new method to StoreController

### CODE REVIEW

**Comment by NazarVynnyk** [comment]
merge branch "nvyn" to develop

File: RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java [hunk 1]
```diff
@@ -67,14 +69,13 @@ public Store getStoreByID(@PathVariable int id, HttpServletResponse response) th
         }
     }
 
-   @PostMapping("/add")
-   @ResponseBody
-   @ResponseStatus(HttpStatus.CREATED)
-   public Store addStore(@RequestBody Store store) {
+  // @PostMapping("/add")
+  @RequestMapping(value = "/add", method = RequestMethod.POST, headers="Accept=application/json")
+  @ResponseStatus(value = HttpStatus.CREATED)
```
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java]: no need to add value if you are not going to specify other properties of the annotation

File: RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java [hunk 2]
```diff
@@ -67,14 +69,13 @@ public Store getStoreByID(@PathVariable int id, HttpServletResponse response) th
         }
     }
 
-   @PostMapping("/add")
-   @ResponseBody
-   @ResponseStatus(HttpStatus.CREATED)
-   public Store addStore(@RequestBody Store store) {
+  // @PostMapping("/add")
```
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java]: Remove commented code

File: RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java [hunk 3]
```diff
@@ -103,4 +104,29 @@ public void deleteStore(@PathVariable int id, HttpServletResponse response) thro
...
+     * @param id current store_id
+     * @param response
+     * @return list of products that sell at the current store
+     * @throws IOException if current store hasn't any product we inform user
+     */
+
+    @GetMapping("/{id}/products")
+    @ResponseBody
+    @ResponseStatus(value = HttpStatus.OK)
+    public List<Product> getAllProducts(@PathVariable int id, HttpServletResponse response) throws IOException {
```
Comment (kucherif) [inline RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java]: Please follwo the same style for annotations. If you are going to use @RequestMapping, please use it everywhere, If you @GetMapping, @Response status, then use those annotataions everywhere. Do not mix old and new style

**Pull request merged**